specification.md

```diff
@@ -129,6 +129,7 @@ A `String` representing a role at the competition.
 
 The specification defines the following roles:
 - `"delegate"`
+- `"trainee_delegate"`
```

Comment:
We already use roles such as `staff-dataentry`, so let's keep the same convention here :)

```suggestion
- `"trainee-delegate"`
```